api/features/permissions.py

```diff
@@ -46,28 +53,66 @@ def has_object_permission(self, request, view, obj):
         return False
 
 
-class FeatureStatePermissions(BasePermission):
+class FeatureStatePermissions(IsAuthenticated):
     def has_permission(self, request, view):
+        action_permission_map = {
+            "list": VIEW_ENVIRONMENT,
+            "create": UPDATE_FEATURE_STATE,
+        }
+        if not super().has_permission(request, view):
```

Comment:
I don't quite get the benefit of sub classing IsAuthenticated if we have to include this line anyway. Surely it's better to just have the IsAuthenticated class in the list or permission_classes on the view?